src/components/file-uploader/FileUploader.jsx

```diff
@@ -0,0 +1,278 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import autoBindReact from 'auto-bind/react';
+import _ from 'lodash';
+import { HelpBlock } from 'react-bootstrap';
+import Dropzone from 'react-dropzone';
+import classNames from 'classnames';
+import FilePreview from '../file-preview';
+import FileUploadPlaceholder from '../file-upload-placeholder';
+import { LoaderContainer } from '../loader';
+import './style.scss';
+
+export default class FileUploader extends React.Component {
+  constructor(props) {
+    super(props);
+    autoBindReact(this);
+
+    this.state = {
+      inProgress: false,
+      error: null,
+    };
+  }
+
+  handleUploadFailed(errorMessage) {
+    this.setState({
+      error: errorMessage,
+      inProgress: false,
+    });
+    this.props.onError(errorMessage);
+  }
+
+  handleUploadSucceeded(fileUrl) {
+    this.props.onUploadSuccess(fileUrl);
+    this.setState({ inProgress: false });
+  }
+
+  validateFileSize(file) {
+    const { localization, maxFileSize } = this.props;
+
+    const { fileMaxSizeError } = localization;
```

Comment:
You can skip this deconstruct and call it like you did before, `localization.something`.
It's safe because it has default value defined in defaultProps.

Or you can deconstruct all and use. (personally, I like `localization.string` better)
Either way, stick to one or another.